docs/install/manual-install.md

```diff
@@ -57,28 +57,27 @@ which it's much simpler to install via a packaging system.
 
 ##### i. MacPorts
 
-{% highlight bash %}
-$ port install gettext jhead libpng openssl p5-locale-gettext p5-perlmagick postgresql91-server
-{% endhighlight %}
+You will need to install gettext, jhead, libpng, openssl (perhaps version 1.1),
+p5-locale-gettext, p5-perlmagick, and postgresql13-server.
```

Comment:
Why has it changed to postgresql13-server?

Reply:
Postgres 9 is no longer supported, best if we keep pace with what the servers are using, which is currently 13.

Reply:
I hadn't realised that. I had got confused by it saying '91' and then '13' which superficially looks like a downgrade.